### ISSUE TITLE
Associer un Hashtag

### ISSUE THREAD

**Issue by EpicKiwi** [EpicKiwi/commissions-bde-cesi-lyon#12]
La fonctionnalité "Associer un Hashtag" permet d'associer une commission à un hashtag.
Le serveur fait alors remonter les différents Tweets, Posts Facebook et Vidéos Youtube portant ce hashtag lors de l'affichage de la commission.
On affiche alors les différents posts sur la page de commission.

Cela permet à la commission de communiquer au travers des réseaux sociaux et de faire remonter la communication sur le site et éventuellement dans la newsletter.

Le hashtag ne peut être associé que si la charge de posts lors de l'association ne représente pas plus de 3 posts durnat la dernière semaine. Cela permet d'éviter qu'une commission ne s'approprie un hashtag trop viral.

**Comment by TngBlt** [comment]
> "Le hashtag ne peut être associé que si la charge de posts lors de l'association ne représente pas plus de 3 posts durant la dernière semaine. Cela permet d'éviter qu'une commission ne s'approprie un hashtag trop viral." 

Tu veux dire que si le hashtag à plus de trois posts, alors ça marche pas ?

**Comment by EpicKiwi** [comment]
Oui, ça met un mot du genre "Ce Hashtag semble trop actif, ce hashtage ne représentera pas l'activité de ta commission" et ça peux proposer des exemples de hastag valides en ajoutant CESi, Lyon ou ce genre de mots d'exemples.
